### DISCUSSION TITLE
YouTube restyling

### DISCUSSION THREAD

**Discussion by chadriae** [Anarios/return-youtube-dislike#718] [General]
Since today YouTube has gotten a restyling of their ui and I can't see the dislike count anymore. Will there be (a possibility for) an update of the extensions? I really got used to having the dislike count back.
<img width="726" alt="Screenshot 2022-08-05 at 19 56 00" src="https://user-images.githubusercontent.com/67370296/183133877-f6c0fe10-d646-42c9-b6b9-74b36573044f.png">

**Comment by cyrildtm** [comment]
I think we need to make a wider update to accommodate most future changes. See #716

**Comment by sy-b** [comment]
A [long] while ago I was experimenting with YT UI change resistance for RYD.
It was a success for 3 desktop + 1 mobile UIs available back then. The 4th one was rare. Next morning I found the 4th one, pasted the script in devtools & nothing happened. Tried on other UIs & got the same result. I knew I did something dumb before going to bed. As usual, I put it on the back burner & forgot.

Fun fact: while trying to fix it, I found that dislikes were still accessible in YT short's accessibility text.

Although I deleted the project folder with other garbage while cleaning my HDD, I should be able to make it again.

The reason I was using devtools' console to test this was because the extension is written in a way to adhere the old UI's structure.
Incorporating the script was like almost rewriting half of the extension's core.

**Reply by sy-b** [reply]
> Incorporating the script was like almost rewriting half of the extension's core.

i.e. breaking lot of things & creating more bugs

**Reply by cyrildtm** [reply]
> > Incorporating the script was like almost rewriting half of the extension's core.
> 
> i.e. breaking lot of things & creating more bugs

I think it's the pain we will have to take. This plugin does not work alone. When youtube changes its behaviors, we have to adapt. It's either making a customized script for every variation of style, or arrive at a universal method that is mostly invariant.

**Comment by sy-b** [comment]
@chadriae  Can you share the html file?
Make sure to **remove your personal info** if present. Or just open in incognito & then save the html.
